accordion-shortcodes.php

```diff
@@ -301,7 +301,7 @@ public function accordion_item_shortcode($atts, $content = null) {
 			$state ? ' data-initialstate="' . $state . '"' : ''
 		);
 
-		$accordion_content = sprintf('<%1$s id="%3$s" class="accordion-content" role="tabpanel" aria-labelledby="%4$s" aria-hidden="true">%2$s</%1$s>',
```

Comment:
I don't see a reason to remove the `aria-labelledby` attribute here. The spec says it's optional, so why not include it?

Reply:
Thanks for the question, Phil. I interpreted this as an optional `aria-labelledby` can be added to the content that is hidden/shown under the button _if_ it also has either `role=region` or is marked up as a `<section>` element (since sections have a native role of region). There is also a note below that line that warns:

> Avoid using the region role in circumstances that create landmark region proliferation, e.g., in an accordion that contains more than approximately 6 panels that can be expanded at the same time.

I've noticed that the plugin allows users to open accordions on page load, and to expand different accordion items at the same time.  Depending on how many accordion tabs are included on a page (some web editors use accordions wisely but not all unfortunately), this could create the landmark proliferation the spec mentions, which means it would be better to avoid using role=region. This means that the aria-labelledby should also be removed, since the role is also removed.